assets/scripts/main.js

```diff
@@ -0,0 +1,72 @@
+const myLibrary = [];
+
+function Book(title, author, page, readed) {
+  if (readed === 'read') {
+    readed = true;
+  } else {
+    readed = false;
+  }
+
+  this.title = title;
+  this.author = author;
+  this.page = page;
+  this.readed = readed;
+}
+
+function addBookToLibrary(book) {
+  myLibrary.push(book);
+}
+
+const bookForm = document.forms['book-form'];
+const bookTable = document.querySelector('#book-table');
+
+bookForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = bookForm.querySelector('#title').value;
+  const author = bookForm.querySelector('#author').value;
+  const page = bookForm.querySelector('#page').value;
+  const readed = bookForm.querySelector('#read').value;
+
+  const tr = document.createElement('tr');
+  const td1 = document.createElement('td');
+  const td2 = document.createElement('td');
+  const td3 = document.createElement('td');
+  const td4 = document.createElement('button');
+  const deleteButton = document.createElement('button');
+
+  td1.textContent = title;
+  td2.textContent = author;
+  td3.textContent = page;
+  td4.textContent = readed;
+
+  deleteButton.textContent = 'Delete';
+  deleteButton.className = 'delete';
+  td4.id = 'readButton';
+
+  tr.appendChild(td1);
+  tr.appendChild(td2);
+  tr.appendChild(td3);
```

Comment:
- [ ] For this piece of code, you are rendering each book in your library app, so it will be appropriate if you create a method for this in order to keep it nice and organized. And also remember that the requirements say a render function should be created to render the arrays of books to the DOM. Here is a [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/render_requirement.png) from Odin.

Secondly, you should try as much as you can to keep your code modular and DRY. The repeated DOM functions can as well be written using the ES6 syntaxes and for loops. For example, this piece of code can be written as the one below - 
```
  const title = bookForm.querySelector('#title').value;
  const author = bookForm.querySelector('#author').value;
  const page = bookForm.querySelector('#page').value;
  const readed = bookForm.querySelector('#read').value;
```

```
let formValues = [];
['#title', '#author', '#page', '#read'].forEach(val => formValues.push(bookForm.querySelector(val).value))
let [title, author, page, readed] = formValues;
```

Reply:
:100: